src/sql_types/src/lib.rs

```diff
@@ -112,6 +114,34 @@ impl SqlType {
             Self::Interval => PostgreSqlType::Interval,
         }
     }
+
+    pub fn is_integer(&self) -> bool {
+        match self {
+            Self::SmallInt(_) | Self::Integer(_) | Self::BigInt(_) => true,
+            _ => false,
+        }
+    }
+
+    pub fn is_float(&self) -> bool {
+        match self {
+            Self::Real | Self::DoublePrecision => true,
+            _ => false,
+        }
+    }
+
+    pub fn is_string(&self) -> bool {
+        match self {
+            Self::VarChar(_) | Self::Char(_) => true,
+            _ => false,
+        }
+    }
+
+    pub fn is_boolean(&self) -> bool {
+        match self {
+            Self::Bool => true,
+            _ => false,
+        }
+    }
```

Comment:
I couldn't find usage of these functions. Do we need them now? If we need them we can add them in PR where we are going to use them.
Could you remove them?